### ISSUE TITLE
PPO Training of Draw-Triangle: Tracking Mismatch and Policy Instability

### ISSUE THREAD

**Issue by Yuan-Xinyi** [haosulab/ManiSkill#1341]
Thank you so much for the work! 

I am training the Draw-Triangle task using the default PPO script parameters, and want to reach the performance demonstrated in the official demo: https://maniskill.readthedocs.io/en/latest/tasks/drawing/index.html. However, the end-effector of Franka is struggling to closely follow the target triangle path. Also, after several epochs, the manipulator flies toward the sky (and the policy diverges).

I am really new to RL, and would appreciate any hints on parameter tuning (e.g., clip_range, learning_rate, entropy_coef). Thank you so much.

**Comment by StoneT2000** [comment]
PPO can't solve the task. Note that in the documentation it doesn't have a dense reward.

The video demo is showing the motion planning generated demo replay. We have only solved it from state info using diffusion policy in the past, there is a baseline for it.
